packages/git/src/browser/history/git-history-widget.tsx

```diff
@@ -26,7 +26,7 @@ import { GIT_HISTORY_ID, GIT_HISTORY_MAX_COUNT, GIT_HISTORY_LABEL } from './git-
 import { GitFileStatus, Git, GitFileChange, Repository } from '../../common';
 import { FileSystem } from '@theia/filesystem/lib/common';
 import { GitDiffContribution } from '../diff/git-diff-contribution';
-import { GitAvatarService } from './git-avatar-service';
+import { AvatarService } from '@theia/scm/lib/browser/avatar-service';
```

Comment:
`ScmAvatarService`

Reply:
The service does not have any dependency on SCM, just like it did not have any dependency on Git. 
However as it's in the package and it is only used by SCM packages I have rename it.

Reply:
Understood, we can pull it out and rename later if it is useful for more generic cases.